inc/commonitilobject.class.php

```diff
@@ -2594,7 +2593,7 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
                         $linkclass->update($data);
                      }
                   }
-                  $linkclass = new $this->supplierlinkclass();
+                  $linkclass = new $item->supplierlinkclass();
```

Comment:
Oh no, one is _supplierlinkclass_, other is _userlinkclass_ it's not the same :p